scripts/error-codes/codes.json

```diff
@@ -253,7 +253,7 @@
   "251": "batch.commit: Cannot commit a batch multiple times.",
   "252": "The `document` global was defined when React was initialized, but is not defined anymore. This can happen in a test environment if a component schedules an update from an asynchronous callback, but the test has already finished running. To solve this, you can either unmount the component at the end of your test (and ensure that any asynchronous operations get canceled in `componentWillUnmount`), or you can change the test itself to be asynchronous.",
   "253": "work.commit(): Cannot commit while already rendering. This likely means you attempted to commit from inside a lifecycle method.",
-  "254": "Element ref was specified as a string (%s) but no owner was set. This could happen for one of the following reasons:\n1. You may be adding a ref to a functional component\n2. You may be adding a ref to a component that was not created inside a component's render method\n3. You have multiple copies of React loaded\nSee https://fb.me/react-refs-must-have-owner for more information.",
```

Comment:
This is autogenerated and corresponds to existing error messages in past versions. Please revert changes to this file.

Reply:
I‘m very sorry that i accidently deleted my fork repository both remote and local,could i close this pull request and open a new one?

Reply:
sure